### PR TITLE
update api for controlling sync state

### PR DESCRIPTION
**Pull request by karissa**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ module.exports = function (osm, media, opts) {
   router.addRoute('GET /sync/destroy', api.syncDestroy.bind(api))
   router.addRoute('GET /sync/join', api.syncJoin.bind(api))
   router.addRoute('GET /sync/leave', api.syncLeave.bind(api))
+  router.addRoute('GET /sync/clear', api.syncClear.bind(api))
```
Comment (noffle) [inline index.js]: What should this do? Could you add a README entry for this route?

File: api.js [hunk 2]
```diff
@@ -327,14 +327,19 @@ Api.prototype.syncPeers = function (req, res, m, q) {
   }
 }
 
+Api.prototype.syncClear = function (req, res) {
+  this.core.sync.clear()
```
Comment (noffle) [inline api.js]: Should this be [`clearState`](https://github.com/digidem/mapeo-core/blob/a77e3b33b462cae92de6d4c1323fcb6b5600fdb8/sync.js#L56)?

File: index.js (hunk 1 above)
Comment (karissa) [reply index.js]: this clears the sync state, which answers your other question on the pr about when does the 'open connections' property get cleared

File: api.js (hunk 2 above)
Comment (karissa) [reply api.js]: yeah

**Pull request merged**